docs/osquery/osquery.asciidoc

```diff
@@ -373,31 +267,61 @@ This integration supports x64 architecture on Windows, MacOS, and Linux platform
 and ARM64 architecture on Linux.
 
 NOTE: The original {filebeat-ref}/filebeat-module-osquery.html[Filebeat Osquery module]
-and the https://docs.elastic.co/en/integrations/osquery[Osquery Log Collection]
+and the https://docs.elastic.co/en/integrations/osquery[Osquery]
 integration collect logs from self-managed Osquery deployments.
 The *Osquery Manager* integration manages Osquery deployments
 and supports running and scheduling queries from {kib}.
 
 [float]
-== Customize Osquery sub-feature privileges
+=== Customize Osquery sub-feature privileges
 
 Depending on your https://www.elastic.co/subscriptions[subscription level],
 you can further customize the sub-feature privileges
 for *Osquery Manager*. These include options to grant specific access for running live queries,
-running saved queries, saving queries, and scheduling queries. For example,
+running saved queries, saving queries, and scheduling packs. For example,
 you can create roles for users who can only run live or saved queries, but who cannot save or schedule queries.
 This is useful for teams who need in-depth and detailed control.
 
 [float]
-== Upgrade Osquery versions
+=== Customize Osquery configuration
+By default, all Osquery Manager integrations share the same osquery configuration. However, you can customize how osquery is configured by editing the Osquery Manager integration for each agent policy
```

Comment:
osquery is lowercase here.  Is that ok?

Reply:
I decided to just be consistent throughout and use caps in all instances.